utilities/mobile_server_pool.py

```diff
@@ -111,7 +109,7 @@ def reserve_node(doc_id, job_name, counter=0):
     try:
         sdk_client.replace(doc_id, doc, cas=curr_cas)
         return True
-    except KeyExistsError as err:
+    except CouchbaseException as err:
```

Comment:
Better to avoid generic exception and catch equivalent of KeyExistsError?

Reply:
no , we do not have